apps/api-wallet/src/app/transactions/transactions.service.ts

```diff
@@ -49,12 +50,33 @@ export class TransactionsService {
     return history;
   }
 
-  async getHistoryByUser(userID: number): Promise<any | undefined> {
+  async getHistoryByUser(
+    userID: number,
+    page: number,
+    limit: number
+  ): Promise<any | undefined> {
+    const startIndex = (page - 1) * limit;
+    const endIndex = page * limit;
+
     const u_history = await this.databaseService.getQueryResult(
```

Comment:
useCamelCase when declaring variables, and be descriptive, we don't know what `u` means

Reply:
done